src/GraphQL/Types/Composite/ComplexGraphType.cs

```diff
@@ -14,13 +14,23 @@ public abstract class ComplexGraphType<TSourceType> : GraphType, IComplexGraphTy
     {
         internal const string ORIGINAL_EXPRESSION_PROPERTY_NAME = nameof(ORIGINAL_EXPRESSION_PROPERTY_NAME);
 
+        private bool _initialized;
+
         /// <inheritdoc/>
         protected ComplexGraphType()
         {
             Description ??= typeof(TSourceType).Description();
             DeprecationReason ??= typeof(TSourceType).ObsoleteMessage();
         }
 
+        /// <inheritdoc/>
+        public override void Initialize(ISchema schema)
+        {
+            if (_initialized)
+                throw new InvalidOperationException($"This graph type '{Name}' has already been initialized.");
```

Comment:
```suggestion
                throw new InvalidOperationException($"This graph type '{Name}' has already been initialized. Make sure that you do not use the same instance of graph type in multiple schemas");
```

`This graph type '{Name}' has already been initialized` message as is does not tell how to fix the problem.